{"name": "Проверка логина с учетной записью пользователя с неверным логином/паролем по очереди и возвращения соответствующей ошибки в теле ответа", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'wrong', 'password': 'password', 'name': 'Username'}"}], "start": 1725772078723, "stop": 1725772078874, "uuid": "175466aa-d61d-4438-b858-e35016bbf2ad", "historyId": "be2fc6a6f4ebdc6b1f867fb9153a5bd6", "testCaseId": "b10a4ce427d230b95d71debe601b33f5", "fullName": "tests.test_login.TestLogin#test_create_user_without_1_required_field", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Home-PC"}, {"name": "thread", "value": "5052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}